### ISSUE TITLE
"Windows installer service could not be accessed"

### ISSUE THREAD

**Issue by EpicLPer** [sandboxie-plus/Sandboxie#1437]
**Describe the problem**
When trying to install iCloud it'll throw an error that the Windows Installer Service cannot be accessed at the "Installing Windows Firewall configuration" step.

**To Reproduce**
1. Download iCloud (Non-Store Version) from https://support.apple.com/de-de/HT204283
2. Try and install it in Sandboxie-Plus

**Screenshots**
![image](https://user-images.githubusercontent.com/8442196/146491906-da15daa6-7af8-4e48-9d3d-229d1f86716c.png)

**Expected behavior**
Installation should finish.

**System details and installed software**
 - What is your Windows edition and version? - Windows 10 x64 21H1
 - What is your current Sandboxie edition and version? - 0.9.8d
 - What was your previous Sandboxie version before the update, if any? - 0.9.3
 - Please mention any security software running in the background, even if the real-time protection is disabled. - Windows Defender

**If you had a compatibility issue or a crash**
 - Please add a download link to the program not working properly in Sandboxie. - https://support.apple.com/de-de/HT204283
 - Please consider to take a log and make it available for analysis: https://git.io/Jwj2y - Can provide later if really needed for this issue.
 - Please attach the actual memory dump and NOT the simple output, which often turns out to be insufficient for a proper analysis. - Can provide later if really needed for this issue.

**Additional context**
-

**Sandboxie configuration**
Default Profile Restrictions, nothing else was changed (New Box was generated for this Install).

**Comment by EpicLPer** [comment]
Turning on "Allow MSIServer to run with a sanboxed system token" does let the installation continue, but will later fail at installing the Bonjour service without any further information other than it "being suspended".

**Comment by isaak654** [comment]
About the Bonjour installation, see https://github.com/sandboxie-plus/Sandboxie/issues/567#issuecomment-817123930

**Comment by EpicLPer** [comment]
> About the Bonjour installation, see [#567 (comment)](https://github.com/sandboxie-plus/Sandboxie/issues/567#issuecomment-817123930)

Yeah I think that was what got Bonjour working then, I tried a bunch of things and probably didn't notice I disabled that at some point :)  
Sadly iCloud still won't install fully tho.

**Comment by isaak654** [comment]
> Sadly iCloud still won't install fully tho.

> later fail at installing the Bonjour service without any further information other than it "being suspended".

Confirmed: 
![bonjour_suspended_instance](https://user-images.githubusercontent.com/12372772/146542171-5348f7f2-afbb-4e46-92bd-f7804eda07e8.png)
